tests/test_runner/test_discover_runner.py

```diff
@@ -223,3 +224,45 @@ def test_excluded_tags_displayed(self):
         with captured_stdout() as stdout:
             runner.build_suite(['test_runner_apps.tagged.tests'])
             self.assertIn('Excluding test tag(s): bar, foo.\n', stdout.getvalue())
+
+    def test_get_databases(self):
```

Comment:
can you split this into one test per suite? cleaner than having a long method with comments, and I don't see any state that is required to carry between the tests